servers/rendering/renderer_rd/effects_rd.h

```diff
@@ -59,6 +59,12 @@
 #include "servers/rendering_server.h"
 
 class EffectsRD {
+	enum ViewMode {
```

Comment:
I would rename MONO to SINGLE, MONO can be misunderstood as normalizing stereo to single channel in audio and its the same here.

Reply:
Hmmm, monoscopic is incredibly common use here, we've been using the term MONO in XR for the past 3 years... I'm happy to change it but this is not audio :)

Reply:
Btw @reduz , going through some older code I was reminded of this comment I put in the code around MONO V.S. STUDIO:  `my son says we should call this EYE_CYCLOPS`

In all seriousness, according to Fredia the term that seems to be used a lot in the XR space nowadays is "pancake mode".

Reply:
I've removed this all together simply opting for switching between the normal shaders and multiview variants of the shaders.